### ISSUE TITLE
Go 32-bit

### ISSUE THREAD

**Issue by eschaton** [jbush001/LispMicrocontroller#3]
It seems like just changing a few parameters could take the design to 32 bit without much of any other change.

**Comment by jbush001** [comment]
Good idea. This wouldn't be too hard. I think the lisp_core verilog module should take the data with as a parameter, so it could be synthesized with different widths depending on application. It might also simplify things to modify the instruction format such that the opcode starts at the lowest significant bit. This would mean programs wouldn't need to be compiled for a specific widths.
